cmd/create.go

```diff
@@ -131,30 +139,62 @@ func newCreateConfig(args []string) createConfig {
 // Prompt the user with value of config members, allowing for interaractive changes.
 // Skipped if not in an interactive terminal (non-TTY), or if --confirm false (agree to
 // all prompts) was set (default).
-func (c createConfig) Prompt() createConfig {
+func (c createConfig) Prompt() (createConfig, error) {
 	if !interactiveTerminal() || !c.Confirm {
 		// Just print the basics if not confirming
 		fmt.Printf("Project path: %v\n", c.Path)
 		fmt.Printf("Function name: %v\n", c.Name)
 		fmt.Printf("Runtime: %v\n", c.Runtime)
 		fmt.Printf("Template: %v\n", c.Template)
-		return c
+		return c, nil
 	}
 
-	var derivedName, derivedPath string
-	for {
-		derivedName, derivedPath = deriveNameAndAbsolutePathFromPath(prompt.ForString("Project path", c.Path, prompt.WithRequired(true)))
-		err := utils.ValidateFunctionName(derivedName)
-		if err == nil {
-			break
-		}
-		fmt.Println("Error:", err)
+	var qs = []*survey.Question{
+		{
+			Name: "path",
+			Prompt: &survey.Input{
+				Message: "Project path:",
+				Default: c.Path,
+			},
+			Validate: func(val interface{}) error {
+				derivedName, _ := deriveNameAndAbsolutePathFromPath(val.(string))
+				return utils.ValidateFunctionName(derivedName)
+			},
+		},
+		{
+			Name: "runtime",
+			Prompt: &survey.Input{
+				Message: "Runtime:",
+				Default: c.Runtime,
+				// TODO add runtime suggestions: https://github.com/AlecAivazis/survey#suggestion-options
```

Comment:
This framework has quite nice suggestions options: https://github.com/AlecAivazis/survey#suggestion-options

So we can then suggest available runtimes and templates in the future.

Reply:
Off topic, but I would like someday to be able to suggest runtimes and templates from remote repositories.